docs/language/README.md

```diff
@@ -201,7 +202,7 @@ A data path can be split with the `split` operator as in
 from PoolOne | split (
   => op1 | op2 | ... ;
   => op1 | op2 | ... ;
-) | merge ts | ...
+) | sort ts | ...
```

Comment:
This change came out of the internal discussion we had after I stumbled onto `merge` not being currently recognized as an explicit operator. After the next GA release goes out, @mccanne, @nwt, and I are due to have a wider chat about implicit/explicit `sort` behaviors, and the topic of implicit/explicit `merge` is related to that. In a preliminary chat, @nwt made the point that one approach we may settle on is that the user may express their desired merge behavior similar to what's shown here via `sort`, at which point our optimizer might be free to _implement_ the sort by potentially sorting the multiple flow graph branches and then merging via the sorted field, but that could be made invisible to the user. Therefore, I've opted to use the explicit `sort` for now so we're not showing an operator that's unavailable to the reader.

Reply:
Update: After a conversation with @mccanne, I've pushed a commit that reverses course here. He felt that the `merge` was communicating something important and that this would be lost if replaced with `sort` as I'd been proposing. Based on this he was in favor of just finishing the add of explicit `merge` to the language in the short term. Then @nwt confirmed in a separate chat that it'd be a bit more complicated than just a one-word addition to the grammar (apparently it would need AST changes, test coverage, etc.), plus a proposal in #2905 would have us taking that back out anyway, so the topic should probably live in our backlog behind other current MVP0 priorities. Therefore I've opened #2906 to fill that role and have put back `merge` in this example and linked to that issue from a "Note:" under the example. If we manage to finish implementing explicit `merge` before we release, great. If not, at least now we've got user guidance to go with the currently-non-working example. And for now the `| sort ts` remains in the ztest so at least the other parts of the example's syntax get some test coverage.